src/components/ui/Modal/Modal.tsx

```diff
@@ -0,0 +1,82 @@
+import { Fragment } from "react";
+import { Dialog, Transition } from "@headlessui/react";
```

Comment:
Why did we add another dependency? Couldn't we use daisy UI for Modal?

Reply:
Bhai daisy UI had predefined styles which was getting difficult to overwrite. HeadlessUI on the other hand is accessible, unstyled, and has a lot of useful components that daisy UI does not have such as popovers, auto suggestion dropdowns. Thats why i decided to switch to HeadlessUI